CondFormats/SiPixelObjects/src/SiPixelCalibConfiguration.cc

```diff
@@ -3,7 +3,7 @@
 #include <cstdlib>
 
 short SiPixelCalibConfiguration::vcalIndexForEvent(const uint32_t & eventnumber) const{
-  uint32_t relative_event = abs(eventnumber-1)%patternSize();
+  uint32_t relative_event = (eventnumber-1)%patternSize();
```

Comment:
so presumably the original intent was not abs(eventnumber-1) but rather to protect against the case of eventnumber=0